packages/table/src/cell/editableCell.tsx

```diff
@@ -40,45 +41,81 @@ export interface IEditableCellProps extends ICellProps {
     onConfirm?: (value: string) => void;
 }
 
-export class EditableCell extends React.Component<IEditableCellProps, {}> {
-    private cellElement: HTMLElement;
+export interface IEditableCellState {
+    /**
+     * Stores the editing state of the cell
+     */
```

Comment:
there's no strong need to document state fields cuz they're only used in the component, and this comment doesn't supply new information given the prop name.